test/plugins/features/movies/index.test.js

```diff
@@ -27,9 +37,31 @@ describe('movies integration', () => {
       expect(response.result.object).to.eql('movie');
     });
 
+    it('movie not found', async () => {
```

Comment:
The describes work is that it should kind of read like a sentence. Here I would say something to the effect of `POST errors when a movie is not found` rather than just `POST movie not found`, pls fix here and elsewhere

Reply:
Gotcha! Will fix, thanks for the clarity!

Reply:
Renamed so that's more like a sentence now